web/src/lib/components/album-page/albums-controls.svelte

```diff
@@ -88,14 +97,35 @@
   }
 
   $: sortIcon = $albumViewSettings.sortOrder === SortOrder.Desc ? mdiArrowDownThin : mdiArrowUpThin;
+
+  const albumFilterNames: { [key in AlbumFilter]: string } = {
+    [AlbumFilter.All]: $t('all'),
+    [AlbumFilter.Owned]: $t('owned'),
+    [AlbumFilter.Shared]: $t('shared'),
+  };
+
+  const albumSortByNames: { [key in AlbumSortBy]: string } = {
+    [AlbumSortBy.Title]: $t('sort_title'),
+    [AlbumSortBy.ItemCount]: $t('sort_items'),
+    [AlbumSortBy.DateModified]: $t('sort_modified'),
+    [AlbumSortBy.DateCreated]: $t('sort_created'),
+    [AlbumSortBy.MostRecentPhoto]: $t('sort_recent'),
+    [AlbumSortBy.OldestPhoto]: $t('sort_oldest'),
+  };
+
+  const albumGroupByNames: { [key in AlbumGroupBy]: string } = {
```

Comment:
All these objects should be reactive

Reply:
Can you please help here, not sure how to fix it, thanks.

Reply:
It's a bit annoying the type can't be set directly, but you could use an IIFE

```ts
$: albumGroupByNames = ((): Record<AlbumGroupBy, string> => {
  return {
    [AlbumGroupBy.None]: $t('group_no'),
    [AlbumGroupBy.Owner]: $t('group_owner'),
    [AlbumGroupBy.Year]: $t('group_year'),
  };
})();
```

Reply:
Thanks, fixing...